src/utilities/loadSampleData.ts

```
@@ -84,198 +84,188 @@ async function formatDatabase(): Promise<void> {
+    process.exit(0);
+  }
+}
+
+/**
+ * Parses a date string and returns a Date object. Returns null if the date is invalid.
+ * @param date - The date string to parse
+ * @returns The parsed Date object or null
+ */
+function parseDate(date: string | number | Date): Date | null {
+  const parsedDate = new Date(date);
+  return isNaN(parsedDate.getTime()) ? null : parsedDate;
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)